### PR TITLE
Support TypeScript 4

### PR DESCRIPTION
**Pull request by jorrit**
I've also updated jest, jest-specific-snapshot and ts-jest. I think my changes can be implemented in a nicer fashion but I am not fluent enough in the TypeScript internals.

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "automock": false,
     "globals": {
         "ts-jest": {
-            "tsConfig": "./tsconfig.json"
+            "tsconfig": "./tsconfig.json"
```
Comment (haneenmahd) [inline jest.config.json:5]: You should use camel case letters

File: src/__tests__/baselines/minification-only/minify-single-line-comments-with-interpolations.ts.baseline [hunk 2]
```diff
@@ -105,16 +105,16 @@ TypeScript before transform:
...
-  const Test5 = styled.div \`width:100%;//\${'red'}\${'blue'}\`;
-  const Test6 = styled.div \`background:url("https://google.com");width:100%;\${'green'}//\${'red'}\${'blue'}\`;
-  const Test7 = styled.div \`background:url("https://google.com");width:\${p => p.props.width};\${'green'}//\${'red'}\${'blue'}
+  const Test1 = styled.div \`width:100%;//\${"red"}\`;
+  const Test2 = styled.div \`width:100%;//\${"red"}\`;
+  const Test3 = styled.div \`width:100%;\${"red"};\`;
+  const Test4 = styled.div \`width:100%;//\${"red"}\`;
+  const Test5 = styled.div \`width:100%;//\${"red"}\${"blue"}\`;
+  const Test6 = styled.div \`background:url("https://google.com");width:100%;\${"green"}//\${"red"}\${"blue"}\`;
+  const Test7 = styled.div \`background:url("https://google.com");width:\${p => p.props.width};\${"green"}//\${"red"}\${"blue"}
```
Comment (haneenmahd) [inline src/__tests__/baselines/minification-only/minify-single-line-comments-with-interpolations.ts.baseline:114]: Looks like you are only changing single quotes to double quotes

**Review by haneenmahd — COMMENTED** [review]
He had replaced single quotes with double quotes.
And Replaced node.getSourceFile().filename with the parameter **sourceFile**.fileName

**Comment by Igorbek** [comment]
@jorrit the issue with TypeScript dependency has now been fixed and released in https://github.com/Igorbek/typescript-plugin-styled-components/releases/tag/1.5.0
Other changes are not required at the moment. If you want to upgrade other dependencies please resolve conflicts.

**Comment by Igorbek** [comment]
Closing as it is addressed in 2b72f1d

**Pull request closed without merging**